spec/Section 5 -- Validation.md

```diff
@@ -455,6 +456,16 @@ SameResponseShape(fieldA, fieldB):
   - If {SameResponseShape(subfieldA, subfieldB)} is false, return false.
 - Return true.
 
+SameStreamDirective(fieldA, fieldB):
+
+- If neither {fieldA} nor {fieldB} has a directive named `stream`.
+  - Return true.
+- If both {fieldA} and {fieldB} have a directive named `stream`.
+  - Let {streamA} be the directive named `stream` on {fieldA}.
+  - Let {streamB} be the directive named `stream` on {fieldB}.
+  - If {streamA} and {streamB} have identical sets of arguments, return true.
```

Comment:
if they have just different initialCount, shouldn't it be considered 'the same' case?

Reply:
This was what was decided based on feedback from Facebook's implementation: https://github.com/graphql/graphql-js/pull/2319#discussion_r362990726

If multiple streams on the same field are needed, one can be aliased. Again, if this blocks certain use cases I think it can be explored in a follow up proposal.